response_operations_ui/__init__.py

```diff
@@ -64,4 +65,6 @@ def user_loader(user_id):
 
     setup_blueprints(app)
 
+    logger.info("App setup complete", config=app_config)
```

Comment:
I don't think we can do this.  Might be worth talking to Dan O'Brian about it.  We're not allowed to log sensitive information and this would log unames/passwords that we use to connect to databases, etc

Reply:
No this just logs the *name* of the config object.

Reply:
got an example for me of what it looks like?  I don't see how that'd be useful to anyone.